Add manual VSCode extension installation info

machineloop opened the pull request:
Thanks for the fantastic work, just wanted to add a few notes on how to manually install the extension to try on multiple Rust projects without using the debugger. Also added a link to the VSCode extension docs. Hopefully this can save someone else a few minutes of time.

Thanks again, this is an extremely exciting first look at where the rust tooling is headed!

nrc commented on README.md:
````diff
@@ -19,3 +19,13 @@ variable to the root of your rls checkout:
 ```  
+## Manually add as regular VSCode extension
+
+If you'd like to test on multiple projects and already have the extension working properly, want to load the extension by default.
````
I think there is a grammar mistake here, needs some words before "want"

machineloop replied:
Thanks, good catch, clarified in additional commit.

nrc commented on README.md:
````diff
@@ -19,3 +19,13 @@ variable to the root of your rls checkout:
+Windows: %USERPROFILE%\.vscode\extensions
+Mac/Linux: $HOME/.vscode/extensions
+```
+You can also add `export RLS_ROOT=/Source/rls` to your bash profile or equivalent. More information available via [VSCode docs](https://code.visualstudio.com/Docs/extensions/example-hello-world#_installing-your-extension-locally).
````
This note on `RLS_ROOT` feels like it belongs to the above section rather than this one.

machineloop replied:
Moved to above section below export snippet

nrc commented on README.md:
```diff
@@ -19,3 +19,13 @@ variable to the root of your rls checkout:
+
+If you'd like to test on multiple projects and already have the extension working properly, want to load the extension by default.
+
+After following the above instructions, and successfully building the extension once, copy the entire `rls_vscode` directory to either:
```
Do I need  to restart VSCode after doing this or does it just detect the change?

machineloop replied:
Yes, thanks, restarting VSCode is necessary, added in commit

machineloop commented:
Thanks for the feedback on the notes, fixed in additional commit. Let me know if you think the wording could use some more work. Thanks again for working on this!

chriskrycho commented:
One thing worth noting—I believe you should be able to symlink it instead of copying it, which would have the upside of not requiring you to copy-and-paste again every time you update the extension. (I've done this with my own small VS Code extension, so I know it works).

machineloop commented:
Thanks @chriskrycho, good point. I've changed line to:
>symlink or copy the `rls_vscode` directory

And also added example unix symlink command:

>For example, to setup a symlink on Mac/Linux: `ln -s /path/to/rls_vscode/ ~/.vscode/extensions/rls_vscode`

Squashed to single commit.

nrc commented:
Thanks!
